deps/rabbit/test/amqp_client_SUITE.erl

```diff
@@ -4193,3 +4196,19 @@ has_local_member(QName) ->
         {error, _} ->
             false
     end.
+
+-spec find_event(Type, Props, Events) -> Ret when
+      Type :: atom(),
+      Props :: proplists:proplist(),
+      Events :: [#event{}],
+      Ret :: {value, #event{}} | false.
+
+find_event(Type, Props, Events) when is_list(Props), is_list(Events) ->
+    lists:search(
+      fun(#event{type = EventType, props = EventProps}) ->
+              Type =:= EventType andalso
+                lists:all(
+                  fun({Key, _Value}) ->
+                          lists:keymember(Key, 1, EventProps)
```

Comment:
This commit removes assertions of the event property values (e.g. `<<"/">>` and `<<"guest">>`).
I'm adding the assertions back in https://github.com/rabbitmq/rabbitmq-server/pull/12302.